### PR TITLE
feat: [CI-3354]: enhanced banner route destination

### PR DESCRIPTION
**Pull request by mtran7**
##### Screenshots:

Without 800px min width
image

With 800px min width
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/10-common/layouts/FeatureBanner.tsx [hunk 1]
```diff
@@ -34,28 +34,42 @@ import css from './layouts.module.scss'
 
 export const BANNER_KEY = 'feature_banner_dismissed'
 
-export const isFeatureLimitBreached = (feature?: CheckFeatureReturn) => {
+export const isFeatureLimitBreached = (feature?: CheckFeatureReturn, includesExceeding = false): boolean => {
```
Comment (mtran7) [inline src/modules/10-common/layouts/FeatureBanner.tsx:37]: Add functionality with backwards compatibility

File: src/modules/10-common/layouts/FeatureBanner.tsx [hunk 2]
```diff
@@ -34,28 +34,42 @@ import css from './layouts.module.scss'
...
+    featureDetail?.limit &&
+    featureDetail.count &&
+    (includesExceeding ? featureDetail.count >= featureDetail.limit : featureDetail.count === featureDetail.limit)
+  )
 }
 
 export const FEATURE_USAGE_WARNING_LIMIT = 90
 
-export const isFeatureWarningActive = (feature?: CheckFeatureReturn) => {
+export const isFeatureWarningActive = (feature?: CheckFeatureReturn, includesLimit = false): boolean => {
```
Comment (mtran7) [inline src/modules/10-common/layouts/FeatureBanner.tsx:48]: Add functionality with backwards compatibility

File: src/modules/10-common/layouts/FeatureBanner.tsx [hunk 3]
```diff
@@ -107,7 +131,7 @@ function getBannerBodyByType({
   }
 
   return (
-    <Layout.Horizontal width="95%" padding={{ left: 'large' }}>
+    <Layout.Horizontal width="95%" padding={{ left: 'large' }} style={{ minWidth: '800px' }}>
```
Comment (nanaxuHarness) [inline src/modules/10-common/layouts/FeatureBanner.tsx:134]: nice enhancement, thx!

**Review by nanaxuHarness — APPROVED** [review]

**Review by vardanbansal-harness — APPROVED** [review]

File: src/modules/10-common/layouts/FeatureBanner.tsx (hunk 1 above)
Comment (chiragp1212) [reply src/modules/10-common/layouts/FeatureBanner.tsx:37]: Can we make new functions for such use cases? It gets tough to manage them in the long run

**Review by nanaxuHarness — APPROVED** [review]

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**